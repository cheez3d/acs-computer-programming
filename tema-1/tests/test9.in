alin mihai tudor damian
644477722233202227774447777844426606283344407288555
mihnea si lucas sunt frati
3
mihnea andrei
lucas matei
frati prieteni
dani vlad octa
