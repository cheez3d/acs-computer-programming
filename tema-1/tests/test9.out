25554446606444#44244408#8836667770326444266
mircea cristian matei paul
26637773344407777444062833444077778866807#7774443383366444
3
3266444 88855523 66622282 
88855523 2
0
0
